### PR TITLE
minor change in view of #199

### PR DESCRIPTION
**Pull request by AKS1996**
struct `VariableInfo` has `is_fixed` variable. confused it earlier with `fixed`

### CODE REVIEW

**Comment by AKS1996** [comment]
added a few more attributes, as I wasn't able to get all of them using 
```
MOI.instantiate(Ipopt.Optimizer, with_bridge_type=Float64)
```

File: src/MOI_wrapper.jl [hunk 1]
```diff
@@ -179,6 +179,56 @@ function MOI.get(model::Optimizer, ::MOI.ListOfConstraints)
...
+    end
+    return indices
+end
+
+
+function MOI.get(
+    model::Ipopt.Optimizer,
+    ::MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}}
+) 
+    indices = MOI.ConstraintIndex{MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}}[]
```
Comment (blegat) [inline src/MOI_wrapper.jl:199]: You can do `MOI.ConstraintIndex{MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}}.(eachindex(model.linear_eq_constraints))`

**Comment by blegat** [comment]
Please add a test as suggested in https://github.com/jump-dev/Ipopt.jl/pull/200#pullrequestreview-421292620

File: src/MOI_wrapper.jl (hunk 1 above)
Comment (AKS1996) [reply src/MOI_wrapper.jl:199]: added

**Comment by AKS1996** [comment]
trying to add but it [breaks at this test](https://github.com/jump-dev/MathOptInterface.jl/blob/master/src/Test/modellike.jl#L496). will it require adding methods to support vector constraints in Ipopt?

**Comment by blegat** [comment]
> will it require adding methods to support vector constraints in Ipopt?

No just use `MOIB.instantiate(Ipopt.Optimizer, with_bridge_type=Float64)` instead of `optimizer`

**Comment by AKS1996** [comment]
works now!

File: src/MOI_wrapper.jl [hunk 2]
```diff
@@ -136,6 +136,13 @@ MOI.get(model::Optimizer, ::MOI.ObjectiveFunctionType) = typeof(model.objective)
 
 MOI.get(model::Optimizer, ::MOI.NumberOfVariables) = length(model.variable_info)
 
+MOI.get(model::Optimizer, ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.LessThan{Float64}}) = length(model.linear_le_constraints)
+MOI.get(model::Optimizer, ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}}) = length(model.linear_eq_constraints)
+MOI.get(model::Optimizer, ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}}) = length(model.linear_ge_constraints)
+MOI.get(model::Optimizer, ::MOI.NumberOfConstraints{MOI.SingleVariable, MOI.LessThan{Float64}}) = length(filter(e -> e.has_upper_bound, model.variable_info))
```
Comment (blegat) [inline src/MOI_wrapper.jl:142]: Use `count` instead

File: test/MOI_wrapper.jl [hunk 3]
```diff
@@ -95,3 +95,7 @@ MOI.empty!(optimizer)
 @testset "MOI NLP tests" begin
     MOIT.nlptest(optimizer, config)
 end
+
+@testset "Testing attributes" begin
```
Comment (blegat) [inline test/MOI_wrapper.jl:99]: `Testing getters` would better describe it imo

Comment (AKS1996) [reply test/MOI_wrapper.jl]: changed

File: src/MOI_wrapper.jl (hunk 2 above)
Comment (AKS1996) [reply src/MOI_wrapper.jl]: changed to count

**Pull request merged**